### PR TITLE
edit PS dataset

### PR DESCRIPTION
**Pull request by Gianni-Pucillo**
implementation & testing have been completed.

### CODE REVIEW

**Comment by Gianni-Pucillo** [comment]
This time I haven't forget the '-s' switch thus I don't know why I'm having still this issue with the ECA.

**Comment by Gianni-Pucillo** [comment]
Furthermore I'm going to investigate about why there is no more "new connection" in Theia ran from Ubuntu Virtual Machine. Any suggestion?

File: src/service/DatasetEditManager.ts [hunk 1]
```diff
@@ -55,14 +57,22 @@ export class DatasetEditManager {
...
             vscode.commands.registerCommand(
-                "zosexplorer.editMember",
+                "zosexplorer.edit",
                 async (arg) => {
+                    let passedMember: Member = { name: ""};
+                    const DSO: string = arg.dataset.dataSetOrganization;
+                    if (DSO === DSOrg.PO) {
+                        passedMember = arg.member;
+                    } else {
+                        passedMember.name = arg.dataset.name;
```
Comment (shciubrcom) [inline src/service/DatasetEditManager.ts:64]: Assignment dataset name to member seems wrong. Please make this solution more obvious.

File: src/service/DatasetEditManager.ts [hunk 2]
```diff
@@ -55,14 +57,22 @@ export class DatasetEditManager {
...
+                    }
                     try {
                         await this.editMember(
                             arg.host,
                             arg.dataset,
-                            arg.member,
+                            passedMember,
                             dataProvider,
+                            DSO,
```
Comment (shciubrcom) [inline src/service/DatasetEditManager.ts]: Why `editMember` function should depend on DSO?
Please consider special method for dataset editing.

**Review by shciubrcom — CHANGES_REQUESTED** [review]
Please provide unit tests from these changes.

File: src/service/DatasetEditManager.ts (hunk 2 above)
Comment (Gianni-Pucillo) [reply src/service/DatasetEditManager.ts]: method sign reverted to the original.

File: src/service/DatasetEditManager.ts (hunk 1 above)
Comment (Gianni-Pucillo) [reply src/service/DatasetEditManager.ts:64]: Slightly modified the above suggested review: please let me know.

File: src/service/DatasetEditManager.ts [hunk 3]
```diff
@@ -183,7 +191,7 @@ export class DatasetEditManager {
     ) {
         const content = await this.datasetService.getContent(
             host,
-            `${dataset.name}(${member.name})`,
+            ((dataset.dataSetOrganization === DSOrg.PS) ? `${dataset.name}(${member.name})` : `${dataset.name}`),
```
Comment (shciubrcom) [inline src/service/DatasetEditManager.ts:194]: Edit member shouldn't edit datasets.

Comment (Gianni-Pucillo) [reply src/service/DatasetEditManager.ts:194]: This is the US target. Or you are meaning that I have to clone editMember() method?
Thanks.

**Pull request closed without merging**